lib/view_component/polymorphic_slots.rb

```diff
@@ -42,6 +42,10 @@ def register_polymorphic_slot(slot_name, types, collection:)
           define_method(getter_name) do
             get_slot(slot_name)
           end
+
+          define_method("#{getter_name}?") do
+            get_slot(slot_name).present?
```

Comment:
```suggestion
            public_send(getter_name).present?
```

Reply:
This way, if the implementation for the polymorphic slot ever changes internally (or is even overridden externally), we'll be calling that instead.